### ISSUE TITLE
Accuracy of snapshots

### ISSUE THREAD

**Issue by brentcty-2020** [NVIDIA/DeepLearningExamples#1052]
Hello, how can I test the loss accuracy of a checkpoint trained with FastPitch?

Also, if I train FastPitch, is it also necessary to train WaveGlow for the entire end-to-end to work?

Thanks.

**Comment by alancucki** [comment]
Hi @brentcty-2020,

what do you mean by "loss accuracy"? If you'd like to know, if the loss for a custom dataset is low enough, that's a wrong metric to look at. The range of loss values observed during training will depend on the difficulty of the dataset, quality of the audio, text processing, etc.

As for WaveGlow, you can start with a [pre-trained model](https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/SpeechSynthesis/FastPitch/scripts/download_waveglow.sh).
